### PR TITLE
Add instructions for installing the pre-commit hooks

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
...
-1. pip install -e . --no-deps --force-reinstall
-1. pytest -n auto -vrsx --cov=conda_lock tests
\ No newline at end of file
+1. `mamba create -n test pip pytest-cov pytest-xdist pip`
+1. `conda activate test`
+1. `python -m pip install -r requirements.txt`
+1. `python -m pip install -r requirements-dev.txt`
+1. `pip install -e . --no-deps --force-reinstall`
+1. `pytest -n auto -vrsx --cov=conda_lock tests`
+1. `pre-commit`
```
Comment (mariusvniekerk) [inline CONTRIBUTING.md:13]: Shouldn't this be pre-commit install?

Comment (ericdill) [reply CONTRIBUTING.md:13]: Maybe? I thought it installed if you just ran pre-commit

Comment (mariusvniekerk) [reply CONTRIBUTING.md:13]: Nope that just runs the default config.

Comment (mariusvniekerk) [inline CONTRIBUTING.md:13]:
```suggestion
Whilst not strictly necessary; the CI run using github actions will run pre-commit -- so in order to reduce development friction you may want to install and activate[ pre-commit
](https://pre-commit.com/)
1. `pre-commit install`
```

**Pull request merged**